_layouts/base.html

```diff
@@ -101,16 +101,24 @@
 </head>
 
 <body>
+<!--
```

Comment:
This block is intended for loading jQuery before Google Tag Manager loads, however, it is not currently in use so could be deleted for the time being.